tests/reset_workflow.go

```diff
@@ -344,6 +344,14 @@ func (s *FunctionalSuite) TestResetWorkflowAfterTimeout() {
 	8 WorkflowExecutionCompleted`, events)
 }
 
+func (s *FunctionalSuite) TestResetWorkflow_ExcludeNoneReapplyDefault() {
+	t := resetTest{
+		FunctionalSuite: s,
+		tv:              testvars.New(s.T()),
+	}
+	t.run()
+}
+
```

Comment:
I would just remove explicit set in next test (line 360).

Reply:
Also passing `nil` to `reapplyExcludeTypes` is closer to real usage sceanrio.

Reply:
OK, I'll merge this now but am considering making a follow-on PR to improve this test.